content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -172,9 +170,6 @@ There are a few efforts in progress that will help simplify the DNS story:
   Kubernetes. There is also work in progress to build layers of `MCS` support
   into Istio, which would allow Istio to work with any cloud vendor `MCS`
   controller or even act as the `MCS` controller for the entire mesh.
-
-While `Admiral` is available today, the Istio and Kubernetes communities are
```

Comment:
Why remove this last sentence and not just update it?

Reply:
My reasoning is that the above sentences already make it clear that both Istio and Kubernetes are actively building solutions, and I don't think we really want to over hype Admiral, since it's not really a great solution.

Reply:
ok I'm fine removing it.